x-pack/plugins/actions/README.md

```diff
@@ -106,27 +106,27 @@ Built-In-Actions are configured using the _xpack.actions_ namespoace under _kiba
 | Namespaced Key                         | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Type          |
 | -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
 | _xpack.actions._**enabled**            | Feature toggle which enabled Actions in Kibana.                                                                                                                                                                                                                                                                                                                                                                                                                               | boolean       |
-| _xpack.actions._**whitelistedHosts**   | Which _hostnames_ are whitelisted for the Built-In-Action? This list should contain hostnames of every external service you wish to interact with using Webhooks, Email or any other built in Action. Note that you may use the string "\*" in place of a specific hostname to enable Kibana to target any URL, but keep in mind the potential use of such a feature to execute [SSRF](https://www.owasp.org/index.php/Server_Side_Request_Forgery) attacks from your server. | Array<String> |
+| _xpack.actions._**hostsAllowList**   | Which _hostnames_ are alloed for the Built-In-Action? This list should contain hostnames of every external service you wish to interact with using Webhooks, Email or any other built in Action. Note that you may use the string "\*" in place of a specific hostname to enable Kibana to target any URL, but keep in mind the potential use of such a feature to execute [SSRF](https://www.owasp.org/index.php/Server_Side_Request_Forgery) attacks from your server. | Array<String> |
```

Comment:
There something about `hostsAllowList` that does sounds quite right, though I think it's technically fine.

Comparing to our other settings, the verbs seem to be in the past tense (odd - never thought about that before!), so perhaps `hostsAllowedList` would be better.  

We also don't use "list" in the config name, instead we seem to plural-ize the relevant word, so perhaps `hostsAllowed` would be better?

Although we are moving from "whitelist" to "allow list", so maybe there's a good reason to always use "allowList" in the name?  Another potential would then be `allowListedHosts`, but that seems not great - I think I mentally accept "whitelisted" as a verb, whereas "allowListed" seems weird.

We also seem to put the verb before the noun, so perhaps `allowedHosts` would be better?

Since there are *sooo* many hits here, I guess we should all agree on the name :-)

Reply:
I like `allowedHosts` - sounds very good. I directed here by the other teams renaming PRs - `allowList` used in a few places, but I think we can choose here our own name for allowed hosts list.